codec-registration-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/CodecRegistrationCheckerCompilerPlugin.scala

```diff
@@ -63,10 +75,12 @@ class CodecRegistrationCheckerCompilerPlugin(override val global: Global) extend
         false
     }
   }
-  override val optionsHelp: Option[String] = Some("""
+  override val optionsHelp: Option[String] = Some(s"""
       |. - directory where cache file will be saved, required
-      |--disable - disables the plugin
+      |$disableFlag - disables the plugin
+      |$sourceCodeDirectoryFlag - path of the source code directory, which has to be checked with this plugin
```

Comment:
Hmmm `src/main/scala/hydra`, or rather `services/src/main/scala/hydra`?

As top-level `src/main/scala/` does not exist:

![image](https://user-images.githubusercontent.com/3383210/174327688-f2cca2ef-cbb9-44a2-b610-c8075a2d3911.png)

Reply:
As a small improvement here, I'd extract the path to Scala source code directory programatically from sbt, shouldn't be too difficult (rather than hardcoding `src/main/scala`), let's then think what to do next

Reply:
done